index.js

```diff
@@ -0,0 +1,79 @@
+console.log("Hello world");
+//task one
+let yearOfBirth = 1989;
+let yearFuture=2027;
+let age = yearFuture - yearOfBirth;
+console.log(`You will be  ${age}  years old in  ${yearFuture}  .`);
+//task two
+let dogYearOfBirth = 2015;  
+let dogYearFuture = 2027;   
+let dogYear = dogYearFuture - dogYearOfBirth; // Calculate the age in human years
+
+// Step 2: Add the extra variable
+let shouldShowResultInDogYears = true;  // Change this to false to show result in human years
+
+// Step 3: Calculate the age in dog years if needed
+let ageInDogYears = dogYear * 7;
+
+// Step 4: Log the result based on the value of shouldShowResultInDogYears
+if (shouldShowResultInDogYears) {
+  console.log("Your dog will be " + ageInDogYears + " dog years old in " + dogYearFuture + ".");
+} else {
+  console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture + ".");
+}
+//task 3
+// Function to calculate house price
+function calculateHousePrice(width, depth, height, gardenSize) {
+    const volumeInMeters = width * depth * height;
+    const housePrice = volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+    return housePrice;
+  }
+  
+  // Peter's house details
+  const peterWidth = 8;
+  const peterDepth = 10;
+  const peterHeight = 10;
+  const peterGardenSize = 100;
+  const peterActualCost = 2500000;
```

Comment:
The naming of some of those variables can be improved upon, by adding `House` between `peter` and the metric. Eg. `peterHouseWidth`.
You could also use an object, if you find it cleaner.